### PR TITLE
Use the new formatted issue templates

### PR DESCRIPTION
**Pull request by nicolo-ribaudo**
You can test this at https://github.com/nicolo-ribaudo/pdf.js/issues, and https://github.com/nicolo-ribaudo/pdf.js/issues/1 is an example of the generated issue. It's unfortunate that all the small questions (such as "Operating system and its version") get their own header, but hopefully it will help with getting all the necessary info filled in properly.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.yml
Comment (Snuffleupagus) [inline .github/ISSUE_TEMPLATE/bug.yml:28]: Since users often report bugs for very old versions, can we please also add a new *required* Yes/No question along the following lines (and perhaps with a link to https://github.com/mozilla/pdf.js#online-demo)?
`Is the bug present in the latest PDF.js version?`

File: .github/ISSUE_TEMPLATE/config.yml
Comment (Snuffleupagus) [inline .github/ISSUE_TEMPLATE/config.yml:7]: Could we link to https://bugzilla.mozilla.org/enter_bug.cgi?product=Firefox&component=PDF%20Viewer&groups=firefox-core-security instead, since that seems to prefill the Security checkbox?

File: .github/ISSUE_TEMPLATE/feature.yml
Comment (Snuffleupagus) [inline .github/ISSUE_TEMPLATE/feature.yml:4]: Given that the Firefox PDF Viewer is the primary development target, how about adding a *required* Yes/No question at the top to help highlight that?
`Is the feature relevant to the Firefox PDF Viewer?`

**Comment by nicolo-ribaudo** [comment]
It seems like there is no way to have a yes/no required option whose default value is "unselected". I made the questions about "does this happen in the latest version" and "is this relevant to Firefox" default to no.

**Review by Snuffleupagus — APPROVED** [review]
This looks great to me, thank you for doing this!

(Not landing this right away, simply to give other core contributors a chance to comment.)

**Comment by timvandermeij** [comment]
I have requested a review from myself and @calixteman as additional sign-off on this. I'll try to have this reviewed tomorrow.

**Comment by marco-c** [comment]
LGTM!
Question: should we have "Is the bug present in the latest PDF.js version?" be "Yes" by default since that should be the most common answer?

**Review by marco-c — APPROVED** [review]

**Review by calixteman — APPROVED** [review]
LGTM. Thank you.

**Review by timvandermeij — APPROVED** [review]
Looks good to me too, and I think I agree with the comment from @marco-c that that's probably a better default value for most issues.

**Comment by nicolo-ribaudo** [comment]
Default updated 👍

**Comment by timvandermeij** [comment]
Now that all four approvals are in and the discussed feedback is addressed, let's do this. This should increase the quality of the bug reports, and if need be we can always tweak it further. Thank you for doing this!

**Pull request merged**
